### DISCUSSION TITLE
[SOLVED:] WHERE CAN I FIND A BETA 'DOCUMENTATION' ?

### DISCUSSION THREAD

**Discussion by typxxi** [meatpiHQ/wican-fw#175] [General]
I have seen the update of the most recent beta from 3.11 to [3.21](https://github.com/meatpiHQ/wican-fw/releases/) with a major improvement I guess if that is implemented well cause there is something mentioned like "car model" which made the impression to me that I could be able to choose "VW" and then "e-golf" and maybe "my 2017 - 2021" to get the dongle setup properly without having to deal that much with all the can bus related stuff.

But there is no link in the software release page available or I have not found one or a description. I had checked before 3.11 and again no documentation I could find easily. Therefore went back to 2.98 stable.

Hope to get an answer soon cause I have to write a guideline for 2 cars and usage within homeassistant. 
thanks

**Comment by jay-oswald** [comment]
@typxxi check my recent PR that just got merged, adding a bunch of profiles in for various cars.

Its pretty common to have little to no documentation for betas because things do change regularly, its often written once its stable and getting released.

But with the new update you will just be able to select a car and have things work with a basic yaml in HA so it will be quite easy, so id defintly be waiting for it to come out properly before you write any guides

**Reply by typxxi** [reply]
Yes, 
I had understood that and documentation was not the right term I meant. I was looking for a short description what model could mean cause I was struggling for over 2 weeks to get the e-GOLF soc, range and odometer values into HA and evcc. 

I have to write my guide for me cause in case I have to redo all these steps and the knowlege I got about CANBUS for a new car.
I learned a lot in those 10 days and it is a guide how to read those documents and get the puzzle right cause the e-GOLF SoC has not really been described anywhere cause I had searched for it. Usually people talk about it as "like the e-Up" but that is not the case as I had to learn the hard way. 

I am describing a way based on firmware 2.98 but I was so surprised about 3.21 that I had instantly asked for "more details" please even though I used "documentation". 
Great progress, I will try that later cause I have also played around with 3.11 last week as my last hope before a finnish guy kicked in and helped me to get it right but based on 2.98 
I had seen the Autopid feature in 3.11 though and had thought it would be pretty usefull but went back to 2.98 that the finnish guy was familiar with. 

Will 3.11 beta be replaced by 3.21 or is it a parallel branch ?

Thanks a lot !

**Reply by meatpiHQ** [reply]
@typxxi I saw you other post, I'll add the e-Golf to the car profiles and give you some pointers on how to set it up, it very simple in the latest update.
